packages/astro/src/core/errors/errors-data.ts

```diff
@@ -733,8 +733,9 @@ export const ResponseSentError = {
  */
 export const MiddlewareNoDataOrNextCalled = {
 	name: 'MiddlewareNoDataOrNextCalled',
-	title: "The middleware didn't return a response or call `next`.",
-	message: 'The middleware needs to either return a `Response` object or call the `next` function.',
+	title: "The middleware didn't return a response or call and return `next`.",
```

Comment:
```suggestion
	title: "The middleware didn't return a `Response`.",
```
The title as written is fine, BUT, I'm wondering whether actually saying this in the message is more helpful. The problem is there's no Response returned (however that gets returned, either directly or by calling `next()`). And I think this could actually be useful when combined with my suggestion for the next line, which really emphasizes the need for A RESPONSE (even if it's the one from `next`)